chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -905,62 +904,62 @@ public static DataSet mathFunction(final DataSet function1, final DataSet functi
 
     public static DataSet mathFunction(final DataSet function, final double value, final MathOp op) {
         final String functionName = op.getTag() + "(" + function.getName() + ")";
-        final double[] y = values(DIM_Y, function);
+        final double[] y = function.getValues(DIM_Y);
         final double[] eyn = Arrays.copyOf(errors(function, EYN), y.length);
         final double[] eyp = Arrays.copyOf(errors(function, EYP), y.length);
 
         final int ncount = function.getDataCount();
         switch (op) {
         case ADD:
-            return new DoubleErrorDataSet(functionName, values(DIM_X, function), ArrayMath.add(y, value), eyn, eyp,
+            return new DoubleErrorDataSet(functionName, function.getValues(DIM_X), ArrayMath.add(y, value), eyn, eyp,
                     ncount, true);
         case SUBTRACT:
-            return new DoubleErrorDataSet(functionName, values(DIM_X, function), ArrayMath.subtract(y, value), eyn, eyp,
+            return new DoubleErrorDataSet(functionName, function.getValues(DIM_X), ArrayMath.subtract(y, value), eyn, eyp,
                     ncount, true);
         case MULTIPLY:
-            return new DoubleErrorDataSet(functionName, values(DIM_X, function), ArrayMath.multiply(y, value),
+            return new DoubleErrorDataSet(functionName, function.getValues(DIM_X), ArrayMath.multiply(y, value),
                     ArrayMath.multiply(eyn, value), ArrayMath.multiply(eyp, value), ncount, true);
         case DIVIDE:
-            return new DoubleErrorDataSet(functionName, values(DIM_X, function), ArrayMath.divide(y, value),
+            return new DoubleErrorDataSet(functionName, function.getValues(DIM_X), ArrayMath.divide(y, value),
                     ArrayMath.divide(eyn, value), ArrayMath.divide(eyp, value), ncount, true);
         case SQR:
             for (int i = 0; i < eyn.length; i++) {
-                eyn[i] = 2 * Math.abs(y[i]) * eyn[i];
-                eyp[i] = 2 * Math.abs(y[i]) * eyp[i];
+                eyn[i] = 2 * java.lang.Math.abs(y[i]) * eyn[i];
+                eyp[i] = 2 * java.lang.Math.abs(y[i]) * eyp[i];
             }
-            return new DoubleErrorDataSet(functionName, values(DIM_X, function), ArrayMath.sqr(y), eyn, eyp, ncount,
+            return new DoubleErrorDataSet(functionName, function.getValues(DIM_X), ArrayMath.sqr(y), eyn, eyp, ncount,
                     true);
         case SQRT:
             for (int i = 0; i < eyn.length; i++) {
-                eyn[i] = Math.sqrt(Math.abs(y[i])) * eyn[i];
-                eyp[i] = Math.sqrt(Math.abs(y[i])) * eyp[i];
+                eyn[i] = java.lang.Math.sqrt(java.lang.Math.abs(y[i])) * eyn[i];
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)